docs/_docs/configuration/options.md

```diff
@@ -352,17 +352,26 @@ before your site is served.
     <tr class="setting">
       <td>
         <p class="name"><strong>Base URL</strong></p>
-        <p class="description">Serve the website from the given base URL</p>
+        <p class="description">Serve the website from the given base URL.</p>
       </td>
       <td class="align-center">
         <p><code class="option">baseurl: URL</code></p>
         <p><code class="flag">--baseurl URL</code></p>
       </td>
     </tr>
+    <tr class="setting">
+      <td>
+        <p class="name"><strong>Live Reload</strong></p>
+        <p class="description">Reload the site automatically on the browser when content is edited.</p>
```

Comment:
A small correction here:
*The entire site is not automatically reloaded on the browser. Just the page that has been edited.*
For example, if you are currently at `http://localhost:4000/` and you edit `about.md` that resolves into `http://localhost:4000/about/`, then the *homepage* need not reload automatically..

(Note: *Do confirm above before making a change to this PR..*)